objects/src/block/header.rs

```diff
@@ -275,6 +275,39 @@ impl Deserializable for BlockHeader {
     }
 }
 
+
+
+/// BLOCK NUMBER STRUCT
+
+/// Holds `u32` type  to signify Block Number
+
+
+#[derive(Debug, Eq, PartialEq, Copy, Clone, PartialOrd, Ord)]
```

Comment:
```suggestion
/// A convenience wrapper around a `u32` representing the number of a block.
///
/// Each block has a unique number and block numbers increase monotonically by `1`.
#[derive(Debug, Eq, PartialEq, Copy, Clone, PartialOrd, Ord)]
```

Nit: Remove extra spaces and add some additional info.